### DISCUSSION TITLE
Source for target loudness table?

### DISCUSSION THREAD

**Discussion by thenktor** [complexlogic/rsgain#143] [General]
Hi,

I wonder what is the source of the target loudness table at https://github.com/complexlogic/rsgain?tab=readme-ov-file#target-loudness ? 
Are there any official docs or was it just made by try and error?

I've played an album on Spotify in Firefox and compared it to the same album playing locally at a target loudness of -14 LUFS. Spotify was louder for sure. Then changed the local files to -12 LUFS, which was a better match.
So my impression is that Spotify uses at least -12 LUFS.

**Comment by complexlogic** [comment]
See https://support.spotify.com/us/artists/article/loudness-normalization/
